### DISCUSSION TITLE
found a problem

### DISCUSSION THREAD

**Discussion by Rakib-programmer-23** [microsoft/vscode-cpptools#10776] [General]
c/c++ extension is not installing in vs code

**Comment by sean-mcmanus** [comment]
Our extension doesn't install itself. You can try filing a bug with more info at https://github.com/microsoft/vscode/issues

**Comment by sabershakeri** [comment]
Here are some steps you can try:

First, make sure you have Visual Studio Code installed on your system.

Open Visual Studio Code and click on the Extensions icon on the left-hand side of the screen. It looks like four squares arranged in a grid.

Search for "C/C++" in the Extensions Marketplace search bar.

Click on the "C/C++" extension in the search results and then click on the "Install" button.

Wait for the installation process to complete. This may take a few minutes depending on your internet connection.

Once the installation is complete, you should see a notification in the bottom right corner of the screen indicating that the C/C++ extension has been installed successfully.

If the installation fails or you encounter any issues, here are a few things you can try:

Check that you have the latest version of Visual Studio Code installed.

Try restarting Visual Studio Code and then attempting the installation again.

If you are behind a firewall or proxy server, try disabling it temporarily and then attempting the installation again.

If you are still having issues, you can try installing the C/C++ extension manually. To do this, download the .vsix file for the C/C++ extension from the Visual Studio Code Marketplace website, and then install it using the "Install from VSIX" command in Visual Studio Code.

I hope this helps!
